course7/homeworks/patrickkostjens/complex/complex_stack.hh

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "complex.hh"
+
+#include <memory>
+
+struct ComplexStackNode
```

Comment:
The node is an implementation detail of the stack. It should be a nested, private class of ```ComplexStack```